packages/liveblocks-react/src/room.tsx

```diff
@@ -408,6 +408,48 @@ function makeExtrasForClient<M extends BaseMetadata>(client: OpaqueClient) {
     return;
   }
 
+  const loadRoomThreads = (
```

Comment:
This is now similar to `waitUntilInboxNotificationsLoaded`, right? Should we unify the naming of these helpers if so? Maybe `waitUntilRoomThreadsLoaded(roomId, queryKey)`?